src/sentry/static/sentry/app/components/userListElement.jsx

```diff
@@ -1,40 +0,0 @@
-import PropTypes from 'prop-types';
```

Comment:
this is only used in one place for the foreseeable future, so i just moved it there (`teamMembers.jsx`)

Reply:
👍